whatsapp_fn.py

```diff
@@ -54,6 +58,7 @@ def send_whatsapp_message(url,to,message,preview_url=False):
     return response
 
 def send_whatsapp_message_image(url,to,media_link,media_caption):
+    add_message(to, bot_number, to, media_link, "image", "whatsapp", "outgoing")
```

Comment:
Every message will have two copies. One for sender and one for receiver. Can you also add the second copy ?

Reply:
I can only see one set of message going in each conversation. Not able to see in the code where the copy is created on either side. Can you comment on the relevant section.

Reply:
@satishck1992 as discusses over call I am marking this comment as not relevant.